gatsby_fp/src/styles/facultyprofile.css

```diff
@@ -88,6 +83,13 @@ hr.top-bar{
   padding: 0;
 }
 
+.title > p{
```

Comment:
Although the file isn't perfectly consistent (`hr.top-bar` is clearly a deficiency), it tends to use a space before `{` so let's stick with that. Our JS and PHP style also does space-before-brace. Also on lines 105, 106, 110, etc.